views/lexicon/tabs/pagination.html.twig

```diff
@@ -3,13 +3,13 @@
 {% block tab_content %}
 <div class="pagination">
     <ul class="pull-left"><!--
-        --><li class="is-disabled"><a href="#"><i class="icon-double-angle-left"></i> Previous</a></li><!--
-        --><li><a href="#">1</a></li><!--
-        --><li class="selected"><a href="#">2</a></li><!--
-        --><li><a href="#">3</a></li><!--
-        --><li><a href="#">4</a></li><!--
-        --><li><a href="#">5</a></li><!--
-        --><li><a href="#">Next <i class="icon-double-angle-right"></i></a></li><!--
+        --><li class="is-disabled"><a href="#prev"><i class="icon-double-angle-left"></i> Previous</a></li><!--
```

Comment:
Existing issue but could use spaceless here

Reply:
It's not a helper, it's a raw markup example